plugins/parsers/influx/parser_test.go

```diff
@@ -763,7 +763,7 @@ func TestSeriesParser(t *testing.T) {
 		{
 			name:    "empty",
 			input:   []byte(""),
-			metrics: []telegraf.Metric{},
+			metrics: make([]telegraf.Metric, 0),
```

Comment:
As above, can't you remove the initialization altogether? Same for the instances below.

Reply:
Removed